app/helpers/results_helper.rb

```diff
@@ -74,35 +74,20 @@ def pdf_friendly_paragraphs(text, is_pdf)
     end
   end
 
-  def pdf_friendly_logo(is_pdf)
-    if is_pdf
-      tag.span(class: "gem-c-organisation-logo__name") do
-        tag.ul(style: "list-style-type: none; margin: 0; padding: 0;") do
-          concat tag.li(I18n.t("results.print.legal_aid"))
-          concat tag.li(I18n.t("results.print.agency"))
-        end
-      end
-    else
-      tag.span(class: "gem-c-organisation-logo__name") do
-        concat tag.span(I18n.t("results.print.legal_aid"))
-        concat tag.br(I18n.t("results.print.agency"))
+  def pdf_friendly_logo
```

Comment:
I've removed `is_pdf` here - it was causing the code coverage to drop as we don't actually use this method other than on the download view which always takes `is_pdf: true` - so the `else` block never gets rendered. I couldn't work out the best way to run through it and test it seeing as we don't use it in any other context, so seemed a bit contrived!